sips/sip-170.md

```diff
@@ -0,0 +1,40 @@
+---
+sip: 170
+title: Inflation Diversion for L2 Incentives
+status: Draft
+author: David Goldberg (@gberg1) and Justin Moses (@justinjmoses)
+discussions-to: https://research.synthetix.io
+
+created: 2021-07-27
+---
+
+<!--You can leave these HTML comments in your merged SIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SIPs. Note that an SIP number will be assigned by an editor. When opening a pull request to submit your SIP, please use an abbreviated title in the filename, `sip-draft_title_abbrev.md`. The title should be 44 characters or less.-->
+
+## Simple Summary
+This SIP proposes to divert the 25,000 SNX weekly incentives for L2 from the pDAO (being funded by the sDAO), to instead come from the inflationary SNX supply.
+
+## Abstract
+This SIP will change the funding source of the 25,000 weekly incentives for L2 from the sDAO (via the pDAO) to the inflationary supply of the SNX.
+
+## Motivation
+To move the weekly L2 incentives to come from the inflationary supply and away from the short term interim solution that was the sDAO (via the pDAO).
+
+## Specification and Overview 
+TODO
```

Comment:
The `SynthetixBridgeToOptimism` contract can be added as a `RewardsDistribution.addDistribution()` recipient as it implements `notifyRewardAmount`